<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddEmp</class>
 <widget class="QDialog" name="AddEmp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>543</width>
    <height>322</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Agregar Empleado&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Apellido</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Usuario</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>200</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Contrase√±a</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>70</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Id</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>130</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Salario</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tf_nombre">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>161</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="tf_apellido">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>161</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="tf_pass">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>200</y>
     <width>141</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="tf_usuario">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>200</y>
     <width>161</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="tf_Id">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>70</y>
     <width>141</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>260</y>
     <width>151</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Agregar Empleado</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="sp_salario">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>130</y>
     <width>141</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <double>1000000000.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_existe">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>230</y>
     <width>161</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#f70000;&quot;&gt;.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
